src/kernel/src/fs/mod.rs

```diff
@@ -12,17 +12,22 @@ use param::Param;
 use std::borrow::Borrow;
 use std::collections::HashMap;
 use std::fmt::{Display, Formatter};
+use std::mem::size_of;
 use std::num::{NonZeroI32, TryFromIntError};
 use std::path::PathBuf;
 use std::sync::atomic::{AtomicI32, Ordering};
 use std::sync::Arc;
 use thiserror::Error;
 
+mod dev;
 mod file;
 mod item;
 mod path;
 mod vnode;
 
+//Represents a file descriptor
+pub type Fd = i32;
+
```

Comment:
Better to make it as a tuple struct instead for strong typing.

Reply:
Sorry about that, I though I already did that, I must have used git stash on it or something.